### ISSUE TITLE
BT: botão inativo - um caso isento

### ISSUE THREAD

**Issue by eupsama** [amagovpt/monitor-server#71]
<img width="1892" height="474" alt="button disabled - um caso isento de acordo com as WCAG" src="https://github.com/user-attachments/assets/cdd60f36-c4fb-4991-b03b-98a402692fac" />

Teste feito no formulário existente na página: https://territoriosinteligentes.gov.pt/ajuda-e-contactos  com a v1.0.4 da extensão do AccessMonitor para Chrome.

De acordo com o CS 1.4.3 das WCAG 2.1 os casos dos botões desativados estão isentos de contraste: 

```html
<button ... disabled>
```

[SC 1.4.3 of WCAG 2.1](https://www.w3.org/TR/WCAG21/#contrast-minimum)

> (...)
> Text or images of text that are _**part of an inactive [user interface component](https://www.w3.org/TR/WCAG21/#dfn-user-interface-components)**_, that are [pure decoration](https://www.w3.org/TR/WCAG21/#dfn-pure-decoration), that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

O AccessMonitor assinala erro de contraste neste botão e não está correto: 

<img width="499" height="124" alt="Image" src="https://github.com/user-attachments/assets/1d52e6d5-972f-4e72-9566-b932264ac155" />
